test/fixtures/cookbooks/test/recipes/config_1.rb

```diff
@@ -0,0 +1,28 @@
+haproxy 'package'
+
+haproxy_config_global '' do
```

Comment:
I've aimed at slicing it into config sections. All ears if there's something better.